data_server.py

```diff
@@ -0,0 +1,62 @@
+#Module to load dataset and divide it into train, test and validation sets for X and Y
+
+def Load_dataset(filename):
+
+    import numpy as np
+
+    try:
+        file = open(filename)
+    except:
+        print("Unable to open the file")
+    else:
+        print("File opened")
+
+    X_list = []
+    Y_list = []
+
+    first_line = True
```

Comment:
I suggest trying this instead of one-time flag:
https://stackoverflow.com/questions/4796764/read-file-from-line-2-or-skip-header-row